lesson_09/AnnotationsTask/Test.java

```diff
@@ -0,0 +1,33 @@
+package lesson_09.AnnotationsTask;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Test {
+    public static void main(String[] args) throws NoSuchFieldException, IllegalAccessException, NoSuchMethodException, InvocationTargetException {
```

Comment:
Of course I expect test class where I can run every class and it will check for Version annotaton in it and so on
Ex:
public static void main(String[] args)  {
checkClass(String.class);
checkClass(MyService.class);
}